### PR TITLE
Adds some more guns, and removes some spells/wands from the greater summon guns and magic spell

### PR DESCRIPTION
**Pull request by 1080pCat**
## What Does This PR Do

Adds the following guns to the summon guns;
The scatter laser gun(its pretty much a shotgun lasergun)
The riot c20r donk soft
The dual tube combat shotgun
The experimental KA
The portable emitter(like the borg one but has charge)
The voxxy shipjack spike thrower
The BSG(prebuilt)
The xray gun
The plasma pistol
The stechkin APS pistol(its a three round burst stech)
The overgrown revolver
The PDW9(ERTs commanders gun)
The U-ion disabler
The Lwap
The ak814 
And finally, the arc revolver!

Removes the wand of death, poly morph wand, and blindness spell from summon magic 

## Why It's Good For The Game

Both the pulse rifle and carbine having hit scan now are just far too OP to be so common, this is technically a buff to wizard, but its more so nerfing survivalists, as a pulse rifle/carbine can shoot like 50-100 tazer shots OR pulse rifle hit scan shots to get what they want, or trigger happy crew husking other crewmates that use spells thinking they're the wizard.

The wand of death is just.. why? why does crew need the ability to quite literally one shot anything including the wizard? I get that its summon magic, the wizardd knows what they're getting into, but still, crew can get some powerful weapons and spells still, i dont see why they also need to oneshot things.
Same goes with the poly morph wand, its not oneshotting persay, but its still a 1 click win for crew

And finally, the blindness spell book, because its a AOE thing prone to FF, blinding everyone around you for a annoying amount of time, god forbid they think they're funny and spam the spell in medbay

## Testing

Used the summon guns/magic secret admin command a heap, didn't see the spells or guns i removed from the lists

## Changelog
:cl:
tweak: You now can get more kinds guns from the summon guns spell, see PR for details on what was added
tweak: You can no longer get the wand of death, and blindness spell book, or poly morph wand from the summon magic spell.
/:cl:

### CODE REVIEW

**Comment by lewcc** [comment]
Arguably, if you're removing the wand of death, you'd probably want to remove the wand of polymorph as well, since it can also instantly end a wizard's round.

**Comment by Contrabang** [comment]
I think pulse should stay, wizard gets 2 free points for using the spell, along with a good chance of a blood-thirsty crew with it trying to kill the wizard. I agree with the rest of the stuff + maybe ban polymorph

**Comment by Mitchs98** [comment]
Agree with Contra. You could always dilute the pool a bit more with more guns to make it rarer to obtain rather than removing it outright.

**Comment by 1080pCat** [comment]
alr i'll dilute it a bit over removing the pulses, the death stays gone, and i'll remove poly too

File: code/game/gamemodes/wizard/rightandwrong.dm [hunk 1]
```diff
@@ -55,7 +69,6 @@ GLOBAL_LIST_INIT(summoned_magic, list(
 	/obj/item/spellbook/oneuse/charge,
 	/obj/item/spellbook/oneuse/summonitem,
 	/obj/item/gun/magic/wand,
-	/obj/item/gun/magic/wand/death,
 	/obj/item/gun/magic/wand/resurrection,
 	/obj/item/gun/magic/wand/polymorph,
```
Comment (DGamerL) [inline code/game/gamemodes/wizard/rightandwrong.dm:73]: There's this one aswell

**Review by DGamerL — COMMENTED** [review]
Code looks good, I marked the wand of polymorph if you intend to remove that one aswell

**Review by DGamerL — APPROVED** [review]

File: code/game/gamemodes/wizard/rightandwrong.dm (hunk 1 above)
Comment (1080pCat) [reply code/game/gamemodes/wizard/rightandwrong.dm]: thank you, forgot about this one

**Comment by Henri215** [comment]
Could add the soviet /obj/item/gun/projectile/automatic/ak814 gun too since the ar is included too.

**Review by S34NW — APPROVED** [review]

**Pull request merged**